Apps/pharcsight/arcsight_connector.py

```diff
@@ -21,13 +21,111 @@
 from bs4 import BeautifulSoup
 import json
 
-
 _container_common = {}
 _artifact_common = {}
 
 
-class ArcsightConnector(BaseConnector):
+def _validate_range(input_range, action_result):
```

Comment:
All changes here are 

1.) Removing redundant parenthesis
2.) Change type checks to use `isinstance`
3.) Extract methods from class functions (methods that don't use `self`)

Reply:
Cool. thanks!!